test/e2e/features/profile/scroll/.feature

```diff
@@ -0,0 +1,34 @@
+# Copyright © 2022-2024 IT ENGINEERING MANAGEMENT INC,
+#                       <https://github.com/team113>
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License v3.0 as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License v3.0 for
+# more details.
+#
+# You should have received a copy of the GNU Affero General Public License v3.0
+# along with this program. If not, see
+# <https://www.gnu.org/licenses/agpl-3.0.html>.
+
+Feature: User scrolls MyProfileScrollable
+
+  Scenario: User jump to last item and scrolls from it to the first item
+    Given I am Alice
+
+    When I tap `MenuButton` button
+    And I tap `PublicInformation` button
+    Then I wait until `MyProfileView` is present
+    And I wait until `BigAvatarProfileField` is present
+
+    When I scroll `MenuListView` until `Legal` is present
+    And I tap `Legal` button
+    Then I wait until `BigAvatarProfileField` is absent
+    And I wait until `LegalField` is present
+
+    When I scroll back `MyProfileScrollable` until `BigAvatarProfileField` is present
```

Comment:
Perhaps a `scroll up` would be a more appropriate phrase to use in this step?

```suggestion
    When I scroll up `MyProfileScrollable` until `BigAvatarProfileField` is present
```

Reply:
At first i named it as you suggested, but then I thought that it be better to name it 'back', cause there's possibility for horizontal scroll also.

So, for
Vertical scroll - back is 'up'
Horizontal scroll - back is 'left'

If that's a problem and we won't rely on horizontal scrolls in the future I can correct it.

Reply:
Didn't touch that, should I correct it or leave as it is?

Reply:
@McKoog, it's up to you. I don't really mind whatever way you decide, both seem fine to me after your comment.

Reply:
Ok, then let's leave it as it is